src/components/form/Form.vue

```diff
@@ -161,6 +163,24 @@ export default {
                     if(input.type == 'captcha') {
                         this.containsCaptcha = true;
                     }
+                    if(input.type == 'label') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'text',
+                                data: input,
+                            }
+                        })
+                    }
+                    if(input.type == 'line') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'html',
+                                data: input,
```

Comment:
@Yusuf-00 if there is no input for the data please give it default content for the sake of ease of usage.